test/testdata/rbs/assertions_break.rb

```diff
@@ -7,7 +7,12 @@
 end
 
 while ARGV.any?
-  break ARGV.shift, "foo" #: Array[String]
+  break ARGV.shift, "foo" #: Array[String] # error: Argument does not have asserted type `T::Array[String]`
```

Comment:
Didn't we say in Slack we were keeping the behavior for the array?

https://shopify.slack.com/archives/C014FP56KSP/p1746026275740829?thread_ts=1746023696.850719&cid=C014FP56KSP